addons/mail/static/src/discuss/call/common/call_actions.js

```diff
@@ -40,8 +40,15 @@ callActionsRegistry
     })
     .add("camera-on", {
         condition: (component) => component.rtc,
-        name: (component) =>
-            component.rtc.selfSession.is_camera_on ? _t("Stop camera") : _t("Turn camera on"),
+        unavailable: (component) => component.rtc?.isRemote,
```

Comment:
It's easier to reason with positive properties. So rather calling it `unavailable`, I would name it `available`/`availableWhen`